### DISCUSSION TITLE
Custom mods can receive free pp

### DISCUSSION THREAD

**Discussion by nana123126** [ppy/osu#26749] [Q&A] [answered]
Oops, I accidentally opened the 'Flash Light' mod and adjusted the Flash Light size to 2x (making it Very easy). I thought this score wouldn't be calculated, but now it appears in my 'Best Performance'.
And adjusting this parameter will not result in a loss of FlashLight bonus.

Other similar scores will display: 'This score is still being calculated and will be displayed soon', and it will never be calculated, causing me to trigger this issue.

I hope to fix this issue and then delete this incorrect score.

![(19PB(BT O%WL(W90A`KP7M](https://github.com/ppy/osu/assets/87159802/a5a3e4b7-e335-4a86-a6f6-3199c2891e8a)
![8$HH794Y}L~TN)XAO)))LCJ](https://github.com/ppy/osu/assets/87159802/2e75573c-e0cc-48e6-8d58-6112bec63723)

**Comment by bdach** [comment, marked as answer]
This should be already fixed but the fix is probably not deployed to current lazer live because of other changes. This should be fixed when the real rollout happens.

**Reply by nana123126** [reply]
So is it okay to keep it like this now?

**Reply by Rekunan** [reply]
yes
